Spring.gitignore

```diff
@@ -0,0 +1,54 @@
+# get it from: https://github.com/spring-projects/spring-framework/blob/master/.gitignore
+.gradle
+´build/
+
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# Ignore Spring specifc files
+/build
+bin
+build.sh
+buildSrc/build
+integration-repo
+.settings
+/spring-*/build
+.springBeans
+spring-test/test-output/
+/src/asciidoc/build
+target/
+
+# Eclipse artifacts, including WTP generated manifests
```

Comment:
We maintain the IDE-specific rules under the `Global` folder as this is easier to maintain when there are new releases to IDEs. Could you remove this section and the IDEA section below?